client/tests/integration/triggers/data_trigger.rs

```diff
@@ -0,0 +1,62 @@
+#![allow(clippy::restriction)]
+
+use eyre::Result;
+use iroha_client::client;
+use iroha_data_model::prelude::*;
+use test_network::*;
+
+#[test]
+fn must_execute_both_triggers() -> Result<()> {
+    let (_rt, _peer, mut test_client) = <PeerBuilder>::new().start_with_runtime();
+    wait_for_genesis_committed(&vec![test_client.clone()], 0);
+
+    let account_id: AccountId = "alice@wonderland".parse()?;
+    let asset_definition_id = "rose#wonderland".parse()?;
+    let asset_id = AssetId::new(asset_definition_id, account_id.clone());
+
+    let prev_value = get_asset_value(&mut test_client, asset_id.clone())?;
+
+    let instruction = MintBox::new(1_u32, asset_id.clone());
+    let register_trigger = RegisterBox::new(Trigger::new(
+        "mint_rose_1".parse()?,
+        Action::new(
+            Executable::from(vec![instruction.into()]),
+            Repeats::Indefinitely,
+            account_id.clone(),
+            FilterBox::Data(BySome(DataEntityFilter::ByAccount(BySome(
+                AccountFilter::new(AcceptAll, BySome(AccountEventFilter::ByCreated)),
+            )))),
+        ),
+    ));
+    test_client.submit_blocking(register_trigger)?;
+
+    let instruction = MintBox::new(1_u32, asset_id.clone());
```

Comment:
You can remove this line, cause you already have it above. Just clone instruction in the second trigger